extensions/azdata/src/azdata.ts

```diff
@@ -417,11 +417,11 @@ async function promptToUpdateAzdata(newVersion: string, userRequested: boolean =
  * Prompts user to accept EULA it if was not previously accepted. Stores and returns the user response to EULA prompt.
  * @param memento - memento where the user response is stored.
  * @param userRequested - if true this operation was requested in response to a user issued command, if false it was issued at startup by system
+ * @param onError - if the prompt is being issued to address an Error. This is typically 'true' when Eula needs to be accepted to proceed.
```

Comment:
I would suggest renaming this to `requireConfirmation` so it's clearer what this is actually intended to be used for.